site/get-started.markdown

```diff
@@ -0,0 +1,143 @@
+
+Run `ghci`, which should start a new prompt for you.
+
+Let's do a simple calculation to check Haskell's computing capabilities:
+```
+> 6 + 3^2 * 4
+42
+```
+Hey, they call Haskell lazy, but that was quick!
+
+42 is a nice even number, but what about the even numbers after it? Let's get the first 10 even numbers after 42.
+```
+> take 10 $$ filter even [43..]
+[44,46,48,50,52,54,56,58,60,62]
+```
+
+What is the sum of those?
+```
+> sum it
+530
+```
+**NOTE**: We used a special feature of GHCi here, which is a special `it` variable that remembers the result of the last expression.
+
+Great, you got the first taste of Haskell! Now let's get to running a real program.
+
+## Writing your first Haskell program
+
+In your editor, create a new file named `hievb.hs`.
+
+Write the following in it:
+```
+main = do
+  putStrLn "Hi, everybody!"
+  putStrLn $$ "Pls look at my favorite odd numbers: " ++ filter odd [10..20]
+```
+
+You can now compile it with `ghc`, which will produce `hievb` binary that we will then run:
+```
+> ghc hievb.hs
+> ./hievb
+Hi, everybody!
+Pls look at my favorite odd numbers: [11,13,15,17,19]
+```
+
+There you go, you just wrote a short, polite program in Haskell!
+
+**TIP**: To interpret the source file directly, without producing any build artifacts, you can use the special `runghc` command like this:
+```
+> runghc hievb.hs
+Hi, everybody!
+Pls look at my favorite odd numbers: [11,13,15,17,19]
+```
+
+**GHCI TIP**: You can also load your file directly into `ghci`, which will enable you to play with any functions and other definitions you defined in it. So for our example, we can just load hievb.hs with ghci and then call the function `main` like this:
+```
+> ghci hievb.hs
+... Compiling ...
+> main
+Hi evb!
+```
+
+**PRO TIP**: Once you progress a bit more in your Haskell journey, you will very likely want to use `cabal` or `Stack` for building your projects, and not `ghc` directly.
+
+## Join the community
+
+By joining the Haskell community you will find a great place to ask for help and learn about new developments in the Haskell ecosystem.
+
+Some of the most popular communities are:
+
+ - [r/haskell](https://www.reddit.com/r/haskell/)
+ - [Haskell Discourse](https://discourse.haskell.org/)
+ - [https://wiki.haskell.org/IRC_channel](https://wiki.haskell.org/IRC_channel)
+ 
+We recommend joining right now, and don't be shy to ask for help!
+
+Check [https://www.haskell.org/community](https://www.haskell.org/community) for a full list of Haskell communities.
+
+## Next steps
+
+There is a myth going around that you need Ph.D. in maths to be able to learn Haskell - and while it is true that Haskell in its elegance does often come close to the elegance of mathematics and borrows a lot of concepts from it, you certainly don't need any advanced math knowledge to be proficient with Haskell. Instead, what you need are some good learning resources!
```

Comment:
IMO remove this. This is planting the idea in the heads of people even if you're trying to deny it. People that reached here aren't the ones need convincing anyway.

Reply:
I am ok with removing this if others think the same, but it is a reality that very often Haskell is being perceived this way, I don't think we can do any more harm here, quite the opposite, haskell.org saying that you can do haskell without advanced maths could be a big encorangment to beginners.
Why do you think they don't need convincing? I think a bit convincing can't hurt!

But, as I said, I can certainly easily remove this if others also think it does more harm than good.

Reply:
I expect anyone reading the "get started" guide to be interested in getting started with Haskell, meaning they are have already been convinced to try Haskell out! We should get out of their way and remove all potential obstacles, including them possibly thinking "wait, there are people that think you need a phd to learn Haskell? Maybe this isn't a great idea after all".